### ISSUE TITLE
Home page accessibility issues

### ISSUE THREAD

**Issue by SNefertitiB** [gcivil-nyu-org/team-1-inperson#205]
- VoiceOver doesn't read "Find your next accessible destination". Just tells you there's a search bar there

- VoiceOver doesn't read "recent reports"

- VoiceOver doesn't read the addresses for each report

- if possible, we should add a description for each report/map map: "map with a pin at {address} marking inaccessible {infrastructure type} reported {report date}"
I think html should look something like this:
<img src="/images/kinghenry.jpg" alt="King Henry VIII of England" />

 - Recent report modals: can't reach issue details or updates by tabbing
